cmd/k8sruntime_util.go

```diff
@@ -48,6 +49,23 @@ func setContainers(resource Resource, containers []ContainerV1) Resource {
 	return resource
 }
 
+func setNetworkPolicyFields(nsName string, policyList []string) Resource {
+	obj := &NetworkPolicyV1{}
+	resource := obj.DeepCopyObject()
+	switch t := resource.(type) {
```

Comment:
And then you switch over it to get the concrete `NetPol` type again, that doesn't make sense :)